pom.xml

```diff
@@ -89,24 +89,6 @@
     <build>
         <pluginManagement>
             <plugins>
-                <!-- CODE FORMATTER -->
```

Comment:
Is this on purpose? If so, can you remove all <relative-formatter-path>../../..</relative-formatter-path> as well?